light/store/db/db.go

```diff
@@ -30,13 +30,17 @@ type dbs struct {
 // want to use one DB with many light clients).
```

Comment:
This is somewhat unrelated, but wouldn't it be better to use `dbm.PrefixDB` if the caller wants to store multiple databases in the same underlying database instead of reimplementing that logic here?

Reply:
Sure. Perhaps this is something I can look into in the following PR